services/traction/api/db/models/base.py

```diff
@@ -65,6 +77,29 @@ async def update_by_id(cls, item_id: UUID, values: dict):
                 raise e
 
 
+class TenantScopedModel(BaseModel):
+    tenant_id: uuid.UUID = Field(foreign_key="tenant.id", index=True)
+
+    @classmethod
+    def tenant_select(cls, fallback_tenant_id: UUID = None) -> Select:
+        from api.endpoints.dependencies.tenant_security import get_from_context
```

Comment:
will have to think about this, seems weird to have endpoints dependency in the db. but probably just a refactor when we have other contexts... should leave a TODO here that this is a dependency smell

Reply:
It was just to avoid a circular import, if we moved the tenant_security stuff somewhere else we could probably fix this. 

I'll give it a shot

Reply:
fixed